scio-repl/src/main/scala/com/spotify/scio/repl/IoCommands.scala

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2016 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.repl
+
+import java.io._
+import java.nio.channels.Channels
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions
+import com.google.cloud.dataflow.sdk.util.GcsUtil
+import com.google.cloud.dataflow.sdk.util.GcsUtil.GcsUtilFactory
+import com.google.cloud.dataflow.sdk.util.gcsfs.GcsPath
+import com.spotify.scio.util.ScioUtil
+import kantan.csv.{RowDecoder, RowEncoder}
+import org.apache.avro.file.{DataFileStream, DataFileWriter}
+import org.apache.avro.generic.{GenericDatumReader, GenericDatumWriter, GenericRecord}
+import org.apache.avro.specific.{SpecificDatumReader, SpecificDatumWriter, SpecificRecordBase}
+import org.apache.commons.io.{Charsets, IOUtils}
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
+
+import scala.collection.JavaConverters._
+import scala.reflect.ClassTag
+
+/** Commands for simple file I/O in the REPL. */
+// scalastyle:off regex
+class IoCommands(options: PipelineOptions) {
+
+  private val TEXT = "text/plain"
+  private val BINARY = "application/octet-stream"
+
+  private val gcsUtil: GcsUtil = new GcsUtilFactory().create(options)
+  private val fs = FileSystem.get(new Configuration())
```

Comment:
Something to keep in mind - probably not a problem here - but if someone was to use this functions in a long running service, configuration (on disk/resources) may change - thus you may need to refresh configuration. Just something to remember.